completion/available/flutter.completion.bash

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/bash
+
+if command -v flutter > /dev/null; then
+    eval "$(flutter bash-completion)"
+fi
```

Comment:
Please add a line break after the last line.

Reply:
I am sorry sir its my IDE :man_facepalming:

Reply:
Please check if your IDE has a plugin for [Editorconfig](https://editorconfig.org) - once you install this, it should automatically do these things (like an extra empty line at the end of the file).